File: estacionServicio/src/main/webapp/js/usuario.js
/*
	Clases Usuario
	Atributos:
		Nombre
		Apellido
		Nombre Usuario
		Clave
		Telefono
		Permisos

	escritura de las variables: camelCase
*/


var Usuario = function (nombre, apellido, nombreUsuario, clave, permisos){
	this.usuarioId = 0;
	this.nombre = nombre;
	this.apellido = apellido;
	this.nombreUsuario = nombreUsuario;
	this.clave = clave;
	//this.telefono = telefono;
	this.permisos = permisos;
}

File: src/main/java/com/ggingenieria/estacion/modelos/Usuario.java
package com.ggingenieria.estacion.modelos;


public class Usuario implements Activable {

    private int usuarioId;
    private String nombre;
    private String apellido;
    private String nombreUsuario;
    private String clave;
    private String permisos;
    private boolean activo;

    public int getUsuarioId() {
        return usuarioId;
    }

    public void setUsuarioId(int usuarioId) {
        this.usuarioId = usuarioId;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getApellido() {
        return apellido;
    }

    public void setApellido(String apellido) {
        this.apellido = apellido;
    }

    public String getNombreUsuario() {
        return nombreUsuario;
    }

    public void setNombreUsuario(String nombreUsuario) {
        this.nombreUsuario = nombreUsuario;
    }

    public String getClave() {
        return clave;
    }

    public void setClave(String clave) {
        this.clave = clave;
    }

    public String getPermisos() {
        return permisos;
    }

    public void setPermisos(String permisos) {
        this.permisos = permisos;
    }

    @Override
    public String toString() {
        return "{ usuarioId: " + usuarioId + ", nombre: " + nombre + ", apellido: " + apellido + "}";
    }


    @Override
    public void setActivo(boolean activo) {
        this.activo = activo;
    }

    public boolean isActivo() {
        return activo;
    }
}
